GliaWidgets/Component/Badge/BadgeStyle.swift

```diff
@@ -26,4 +26,29 @@ public struct BadgeStyle {
         self.fontColor = fontColor
         self.backgroundColor = backgroundColor
     }
+
+    /// Apply badge remote configuration
+    public mutating func applyBadgeConfiguration(_ badge: RemoteConfiguration.BadgeStyle?) {
```

Comment:
What is the reason to make method public? We use it only internally

Reply:
Good point! Fixed.